src/select/select.js

```diff
@@ -394,6 +388,18 @@ class Select extends React.Component<PropsT, StatelessStateT> {
       : this.getOptionLabel(option);
   }
 
+  filterOption(option: OptionT, query: string) {
+    if (this.props.filterOption) {
+      return this.props.filterOption(option, query);
+    }
+
+    const label = this.getOptionLabel(option);
+    return (
+      typeof label === 'string' &&
+      label.toLowerCase().indexOf(query.toLowerCase()) >= 0
```

Comment:
can we simply use `String.includes`? https://caniuse.com/#search=includes

Reply:
I suppose it depends if we're trying to support IE11. Did we document our browser support anywhere? I forget.

Reply:
we have babel-polyfill
it should fallback here
https://github.com/zloirock/core-js#ecmascript-string-and-regexp

Reply:
awesome, changed.